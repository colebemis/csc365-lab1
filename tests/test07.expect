Enter query: SAELEE, DANILO
GARTH, JOHN
Enter query: 