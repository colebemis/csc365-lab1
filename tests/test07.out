Enter query: SAELEE, DANILO, 2.85, FALKER, ADOLPH, 54
GARTH, JOHN, 3.14, FALKER, ADOLPH, 0
Enter query: 